### PR TITLE
new suggestions system, scales w/ time

### PR DESCRIPTION
**Pull request by alexdcdc**
i am good at keeping secrets :)

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -23,32 +23,37 @@ async def updateSuggestions():
...
-            if (message.created_at.utcnow()-message.created_at).seconds>(6)*3600 or (message.created_at.utcnow()-message.created_at).days>0:
-                message=bot.suggestQueue.popleft()
-                approvals = get(message.reactions, emoji="✅")
-                denials = get(message.reactions, emoji="❌")
-                if approvals.count>denials.count:
+        for message in bot.suggestQueue:
+            approvalsObject=get(message.reactions, emoji="✅")
+            denialsObject=get(message.reactions, emoji="❌")
+            approvals=approvalsObject.count-(bot.user in set(await approvalsObject.users().flatten())) #gets # of yes reactions that isn't the bot
+            denials=denialsObject.count-(bot.user in set(await denialsObject.users().flatten()))     #gets # of no reactions that isn't the bot
```
Comment (mintexists) [inline bot.py:32]: Why are you not including bots if the bot auto un-reacts? There shouldnt be a bot reaction on it ever

File: bot.py [hunk 2]
```diff
@@ -237,6 +242,7 @@ async def on_member_join(member):
     if member.bot == False:
         await member.guild.get_channel(739647916905332846).send(member.mention + " has joined.\n" + member.guild.get_role(736316470098657342).mention)
         await member.add_roles(member.guild.get_role(743206825176072345), member.guild.get_role(743206597387485324))
+        bot.memberCount+=1
```
Comment (mintexists) [inline bot.py:311]: mite want another one of these for on_member_leave, tho u could remove it if u ignore the not counting bots part as mentioned above

File: bot.py [hunk 3]
```diff
@@ -23,32 +23,37 @@ async def updateSuggestions():
...
                 await message.delete()
         await asyncio.sleep(5)
 
+
 @bot.event
 async def on_ready():
     # Set Status
     await bot.change_presence(activity=discord.Activity(type=discord.ActivityType.watching, name="Republic of United Members"))
     await updateSuggestions()
+    bot.memberCount=len([m for m in bot.get_guild(736306540671271036).members if not m.bot])
```
Comment (mintexists) [inline bot.py:58]: u mite need to await get_guild

File: bot.py (hunk 2 above)
Comment (alexdcdc) [reply bot.py:311]: i’ll fix that

File: bot.py (hunk 3 above)
Comment (alexdcdc) [reply bot.py:58]: ok

File: bot.py (hunk 1 above)
Comment (alexdcdc) [reply bot.py:32]: in case the bot still has the reaction b/c no one has voted

File: bot.py (hunk 3 above)
Comment (alexdcdc) [reply bot.py:58]: can confirm - not a coroutine, dont need to await

**Pull request merged**